File: src/main/java/it/contrader/dto/FolderDTO.java
package it.contrader.dto;

import javax.persistence.EnumType;
import javax.persistence.Enumerated;

import it.contrader.model.User.Usertype;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor

public class FolderDTO {

	private Long idFolder;
	
	private Long idUser;
	
	private String nameFolder;

	

}

File: src/dto/folderdto.ts
export class FolderDTO{

    id? : number;
    nameFolder :string;
    iduser : number;
    
    constructor(id: number, nameFolder: string, iduser: number,) {
        this.iduser = iduser;
        this.id = id;
        this.nameFolder = nameFolder;
    }
}